content/en/docs/setup/platform-setup/kubesphere/index.md

```diff
@@ -34,12 +34,12 @@ Ensure your firewall meets the [port requirements](https://kubesphere.io/docs/v2
 
 ## Provisioning a Kubernetes cluster
 
-1. Download KubeSphere to your Linux machine, it will create a standard Kubernetes cluster (1.15 by default, KubeSphere also supports 1.13 and 1.14):
+1. Download KubeSphere to your Linux machine, it will create a standard Kubernetes cluster (1.16 by default, KubeSphere also supports 1.15 and 1.17):
 
     {{< text bash >}}
-    $ curl -L https://kubesphere.io/download/stable/v2.1.0 > installer.tar.gz
+    $ curl -L https://kubesphere.io/download/stable/latest > installer.tar.gz
```

Comment:
Not sure the best way to handle latest and then specifying a particular version on the `cd` and then there k8s versions.

Reply:
If I look at https://preliminary.istio.io/docs/setup/getting-started/#download (although I know that page is being reworked so it might change soon), but I could see saying something like:
```
Change to the Kubesphere directory. For example, if the created directory is kubesphere-all-v2.2.0, `cd kubesphere-all-v2.2.0`
```
Or maybe you keep the text bash with the cd so they can cut/paste (although if they have completion turned on, it's just type cd<tab> to get it filled out).

Reply:
And then maybe remove the Kubernetes versions since they will change over time and not require updates to this doc.

Reply:
Good suggestion, I have modified it accordingly. Please help to review the updates.